en/includes/guides/authentication/mfa/add-x509-login.md

```diff
@@ -275,119 +307,83 @@ If {{product_name}} is already started, use the [Certificate Validation Manageme
-        - `AuthenticationEndpoint` : This is the URL with the port that is secured with the certificate, 
-            e.g., `https://localhost:8443/x509-certificate-servlet`. 
-            This value will be taken to extract the certificate from the browser by redirecting the user to the specified endpoint. 
-            Update this based on your host name.
-        - `username` : This attribute value will be taken as the authenticated user subject identifier. Update this
-             with any of the certificate attributes, e.g., CN and Email.
+    !!! info "Configuration details"
+        - `name` : Identifies the authenticator. 
+        - `enable`: Enables X.509 certificate authentication.
+        - `AuthenticationEndpoint` : Specifies the URL where the certificate is retrieved from the browser. Update this based on your hostname.
+        - `username` : Defines the certificate attribute used as the authenticated user’s identifier (e.g., CN, Email).
 
     !!! note
-        When X509 authentication is configured as the second authentication
-        step, the certificate will be validated to check whether it is
-        associated with the authenticated user in the first authentication
-        step. For that, the `username` parameter will
-        be used. For that, the authenticated user name considered in the
-        first authentication step will be validated with the certificate
-        attribute in this property.
-    
-        When X509 authentication is configured as the first step, this
-        certificate attribute will be treated as the authenticated user
+
+        When X.509 authentication is used as the second authentication step, the system validates whether the certificate is linked to the user authenticated in the first step. This is done by comparing the `username` parameter with the corresponding attribute in the certificate.  When X509 authentication is configured as the first step, this certificate attribute will be treated as the authenticated user
         subject identifier.
     
-3.  If you are using a user property to store X509 certificate, add the following parameter.
+2.  If you want to store X.509 certificates as user claims, add the following property to the same file:
 
     ``` toml
     [authentication.authenticator.x509_certificate.parameters]
     setClaimURI = "http://wso2.org/claims/userCertificate"
     ```
 
-4.  To enable storing the X509 certificate as a user claim, add the following parameter.
+4.  To enable automatic self-registration for users who authenticate with an X.509 certificate, add the following property to the same file:
 
     ``` toml 
     [authentication.authenticator.x509_certificate.parameters]
     EnforceSelfRegistration = true
     ```
 
-5. Restart the server to apply the changes.
+5. Restart {{product_name}} to apply the changes.
 
-## Add a claim mapping for the certificate
+## Step 6: Add an attribute mapping for the certificate
```

Comment:
Step 6 is only required when the user has enabled storing X.509 certificates as a user attribute